test/simple_test.go

```diff
@@ -28,16 +28,16 @@ func TestGNOI(t *testing.T) {
 		in   proto.Message
 		want string
 	}{{
-		desc: "gpb.Path",
-		in: &gpb.Path{
+		desc: "tpb.Path",
+		in: &tpb.Path{
 			Origin: "oc",
-			Elem:   []*gpb.PathElem{{Name: "interfaces", Key: map[string]string{"name": "Ethernet1/1/0"}}},
+			Elem:   []*tpb.PathElem{{Name: "interfaces", Key: map[string]string{"name": "Ethernet1/1/0"}}},
 		},
 		want: "origin: \"oc\"\nelem: <\n  name: \"interfaces\"\n  key: <\n    key: \"name\"\n    value: \"Ethernet1/1/0\"\n  >\n>\n",
 	}, {
```

Comment:
Would you consider adding the test for system.SetPackage(RemoteDownload with Credentials)? This test requires simple_test.go to import both "gnoi/common" and "gnoi/types".

The code diff is listed below:

```
diff --git a/test/simple_test.go b/test/simple_test.go
index ecc708c..1b8405c 100644
--- a/test/simple_test.go
+++ b/test/simple_test.go
@@ -18,8 +18,10 @@ import (
 	"testing"
 
 	"github.com/golang/protobuf/proto"
-	gpb "github.com/openconfig/gnoi"
 	bgppb "github.com/openconfig/gnoi/bgp"
+	"github.com/openconfig/gnoi/common"
+	gSystem "github.com/openconfig/gnoi/system"
+	gpb "github.com/openconfig/gnoi/types"
 )
 
 func TestGNOI(t *testing.T) {
@@ -34,6 +36,19 @@ func TestGNOI(t *testing.T) {
 			Elem:   []*gpb.PathElem{{Name: "interfaces", Key: map[string]string{"name": "Ethernet1/1/0"}}},
 		},
 		want: "origin: \"oc\"\nelem: <\n  name: \"interfaces\"\n  key: <\n    key: \"name\"\n    value: \"Ethernet1/1/0\"\n  >\n>\n",
+	}, {
+		desc: "system.SetPackage",
+		in: &gSystem.Package{
+			Filename: "filename",
+			RemoteDownload: &gnoi.RemoteDownload{
+				Path:     "foo",
+				Protocol: gnoi.RemoteDownload_SCP,
+				Credentials: &gpb.Credentials{
+					Username: "bar",
+				},
+			},
+		},
+		want: "filename: \"filename\"\nremote_download: <\n  path: \"foo\"\n  protocol: SCP\n  credentials: <\n    username: \"bar\"\n  >\n>\n",
 	}, {
 		desc: "gpb.HashType",
 		in: &gpb.HashType{
```

Reply:
Added this test case, thanks!